recipe/meta.yaml

```diff
@@ -651,6 +652,80 @@ outputs:
         - LICENSE.txt
       summary: C++ libraries for Apache Parquet
 
+  - name: parquet-utils
```

Comment:
Shouldn't the utils be part of `libarrow-all`? The `-all` is pretty unambiguously maximal IMO

Reply:
The `lib` argument is valid, perhaps more than the "`-all` means everything a feedstock provides in one go". 

From my POV, the individual libraries already serve the "I want to compile against _this_ library", so `libarrow-all` is primarily (IMO) just a convenience wrapper.

Reply:
I defer to you